### ISSUE TITLE
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'

### ISSUE THREAD

**Issue by chris-y** [db0/pythorhead#5]
````
Traceback (most recent call last):
  File "lemmythread.py", line 2, in <module>
    from pythorhead import Lemmy
  File "/home/pi/.local/lib/python3.9/site-packages/pythorhead/__init__.py", line 1, in <module>
    from pythorhead.lemmy import Lemmy
  File "/home/pi/.local/lib/python3.9/site-packages/pythorhead/lemmy.py", line 4, in <module>
    from pythorhead.auth import Authentication
  File "/home/pi/.local/lib/python3.9/site-packages/pythorhead/auth.py", line 14, in <module>
    class Authentication(metaclass=Singleton):
  File "/home/pi/.local/lib/python3.9/site-packages/pythorhead/auth.py", line 15, in Authentication
    token: str | None = None
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
````

From Googling this error looks like it comes from not being supported by Python 3.9.2, but the pypi page says it's compatible with >=3.8,<3.11.

**Comment by db0** [comment]
Fixed in #7

**Comment by chris-y** [comment]
Rather hoping the fix would be to change the code so it works on slightly older versions, but I suppose that resolves it!

**Comment by db0** [comment]
I might have to do that myself when it comes down to it. We'll see. @NicKoehler would you be willing to switch the older static typiing  format to support older python? I feel this library is lightwight and dependency-less enough that we don't have to limit it to 3.10+ just for static typing support.

**Comment by db0** [comment]
Fixed in #8 

btw @NicKoehler what's your lemmy account?

**Comment by NicKoehler** [comment]
> btw @NicKoehler what's your lemmy account?

https://feddit.it/u/NicKoehler
